pvnet/models/base_model.py

```diff
@@ -277,19 +281,28 @@ def save_pretrained(
             # Taylor the data config to the model being saved
             minimize_data_config(new_data_config_path, new_data_config_path, self)
 
+        # Get appropriate model card
+        model_name = huggingface_repo.split("/")[1]
+        if model_name == "windnet_india":
+            model_card = "windnet_model_card_template.md"
+        elif model_name == "pvnet_india":
+            model_card = "pvnet_india_model_card_template.md"
+        else:
+            model_card = "model_card_template.md"
+
```

Comment:
hurray! very nice :)

Reply:
Can we rename `model_card_template.md` to something like `uk_regional_model_card_template.md` and `pvnet_india_model_card_template.md` to something like `india_[pv? wind?]_model_card_template.md`? I think we should be very explicit with the names

Reply:
Yep good shout, let me change that